versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5775 @@
+* `wf.test.my_var` - References the `String` input of first call to task `test`
+* `wf.test.results` - References the `File` output of first call to task `test`
+* `wf.test2.my_var` - References the `String` input of second call to task `test`
+* `wf.test2.results` - References the `File` output of second call to task `test`
+* `wf.foobar.results` - References the `File` output of the call to `other.foobar`
+* `wf.foobar.input` - References the `File` input of the call to `other.foobar`
+* `wf.other_workflow` - References the first call to subworkflow `other.other_workflow`
+* `wf.other_workflow.bool` - References the `Boolean` input of the first call to subworkflow `other.other_workflow`
+* `wf.other_workflow.foobar.results` - References the `File` output of the call to `foobar` inside the first call to subworkflow `other.other_workflow`
+* `wf.other_workflow.foobar.input` - References the `File` input of the call to `foobar` inside the first call to subworkflow `other.other_workflow`
+* `wf.other_workflow2` - References the second call to subworkflow `other.other_workflow` (aliased as other_workflow2)
+* `wf.other_workflow2.bool` - References the `Boolean` input of the second call to subworkflow `other.other_workflow`
+* `wf.other_workflow2.foobar.results` - References the `File` output of the call to `foobar` inside the second call to subworkflow `other.other_workflow`
+* `wf.other_workflow2.foobar.input` - References the `File` input of the call to `foobar` inside the second call to subworkflow `other.other_workflow`
+* `wf.arr` - References the `Array[String]` declaration on the workflow
+* `wf.scattered_test` - References the scattered version of `call test`
+* `wf.scattered_test.my_var` - References an `Array[String]` for each element used as `my_var` when running the scattered version of `call test`.
+* `wf.scattered_test.results` - References an `Array[File]` which are the accumulated results from scattering `call test`
+* `wf.scattered_test.1.results` - References an `File` from the second invocation (0-indexed) of `call test` within the scatter block.  This particular invocation used value "b" for `my_var`
+
+A namespaced identifier has the same syntax as a fully-qualified name. It is interpreted as the left-hand side being the name of a namespace and the right-hand side being the name of a workflow, task, or sub-namespace within that namespace.
+
+Consider this workflow:
+
+```wdl
+import "other.wdl" as ns
+
+workflow wf {
+  call ns.ns2.task
+}
+```
+
+Here, `ns.ns2.task` is a namespace identifier (see the [Call Statement](#call-statement) section for more details). Namespace identifiers, like fully-qualified names are left-associative, which means `ns.ns2.task` is interpreted as `((ns.ns2).task)`, which means `ns.ns2` has to resolve to a namespace so that `.task` can be applied. If `ns2` is instead a task definition within `ns`, then the namespaced identifier `ns.ns2.task` is invalid.
+
+### Declarations
```

Comment:
I wish this section could be much higher in the text.  I think the rest of the above text would make more sense if you'd defined what a declaration is, and what does them (tasks and workflows).  That's a big structural issue though, so that might be overwhelming and not worth even more of your effort.

Reply:
I've revised this section. Please read the new version and see if it's more clear.